### PR TITLE
Landing/patcher image fixes

### PR DESCRIPTION
**Pull request by ina-stoyanova**
This PR fixes the landing page(s) for Patcher:
- [x] Images on use case page - removing border around images 
- [x] Adding a new image for the central Patcher landing page
- [x] Updating the site title to be "Patcher | Gruntwork Labs"

### CODE REVIEW

File: _includes/landing-navbar.html [hunk 1]
```diff
@@ -21,8 +21,11 @@
             class="navbar-brand navbar-brand-beta"
             style="padding-left: 0"
             href="{{ landing_page_base_url }}"
-            ><strong>Gruntwork</strong>&nbsp;Labs</a
-          >
+            >
+            {% if page.extra_title %} <strong> {{ page.extra_title }} </strong> &nbsp;|&nbsp; Gruntwork&nbsp;Labs
+            {% else %} <strong>Gruntwork</strong>&nbsp;Labs
+            {% endif %}
```
Comment (brikis98) [inline _includes/landing-navbar.html:27]: Here's a slightly tweaked layout for the text to consider:

Screen Shot 2021-08-18 at 9 42 36 AM

This way, we get more focus on the name of the product, Patcher, as it's now `font-size: 50px`, and stands out more from the rest.

File: pages/landing/patcher/index.html [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
+extra_title: Patcher
+
+title: "Patcher | Gruntwork Labs"
+title_text: Keep your code up to date, even with breaking changes.
 excerpt: Patcher automatically creates pull requests to keep your dependencies up to date, updating not only the version numbers, but also patching your code to handle backwards incompatible changes.
 permalink: /patcher/
 
 hero:
-  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  image: /assets/img/landing-page-patcher/patcher.png
```
Comment (brikis98) [inline pages/landing/patcher/index.html:18]: Hm, this image ends up being a bit blurry, and perhaps even a little misleading. It makes Patcher seem like a purely CLI tool, whereas it's really more of a SaaS app that has a UI and opens PRs. Any other ideas for images?

Comment (ina-stoyanova) [reply pages/landing/patcher/index.html:18]: Hm, I was struggling to get anything else out of my hands yesterday..... I have another idea today, but let's see if I can make it work better. Thank you for the feedback - I think you're right, that it shows only the CLI side.

File: _includes/landing-navbar.html (hunk 1 above)
Comment (ina-stoyanova) [reply _includes/landing-navbar.html:27]: Deal! It does differ from all the other landing pages, but since it's not a service, but more a standalone product, I think it's fine.

**Comment by ina-stoyanova** [comment]
@brikis98 good for another look. I tried to make the image better and to explain more, but I'm pretty sure @ebeneliason might be the best person for that! LMK what you think!

**Comment by brikis98** [comment]
> @brikis98 good for another look. I tried to make the image better and to explain more, but I'm pretty sure @ebeneliason might be the best person for that! LMK what you think!

Ohhh, using the version numbers / SemVer is a great idea! I took it and combined it with an image we had of Grunty "juggling" cubes to get a slightly clearer version:

grunty-cubes-version

It has a transparent background, which might make it hard to see, so here's a screenshot of what it looks like in the page:

Screen Shot 2021-08-19 at 10 49 53 AM

Note that to look a bit better with this image, I also made a few minor tweaks to the name:

- Set `font-size: 40px` on the "Patcher" name
- Set `font-size: 15px` and `line-height: 15px` on the "by Gruntwork" text.
- Set `padding-top: 15px` and `padding-left: 20px` on the `div` with class `main_title_patcher`.

**Comment by brikis98** [comment]
I also came across this, which I think is a pretty hilarious way to show the value of Patcher: https://miro.medium.com/max/2800/1*8rSaJAbeeI9wbLTN5gFI6Q.png

No idea where to incorporate it on the page, so just sharing for fun now :)

**Comment by ina-stoyanova** [comment]
Oh, I love the idea with the cubes ❤️!! I'll make the changes to the name too!

**Comment by ina-stoyanova** [comment]
Changes incorporated. I think it looks great, so thanks for all the help!

File: _includes/landing-navbar.html [hunk 3]
```diff
@@ -21,8 +21,8 @@
             class="navbar-brand navbar-brand-beta"
             style="padding-left: 0"
             href="{{ landing_page_base_url }}"
-            ><strong>Gruntwork</strong>&nbsp;Labs</a
-          >
+            ><strong>Gruntwork</strong>&nbsp&nbsp;Labs
```
Comment (brikis98) [inline _includes/landing-navbar.html:24]: NIT: Is this change still needed in this PR?

**Review by brikis98 — COMMENTED** [review]
One tiny NIT and this is good to go!

**Comment by brikis98** [comment]
> Oh, I love the idea with the cubes ❤️!! I'll make the changes to the name too!

The version numbers are your ideas. I just remembered which Grunty images we have 😁

File: _includes/landing-navbar.html (hunk 3 above)
Comment (ina-stoyanova) [reply _includes/landing-navbar.html:24]: Oh, that must be putting it on one line. I don't think it's got a place in this PR! Thanks for calling it out!

**Review by brikis98 — APPROVED** [review]
LGTM! Let's merge and release.

**Comment by ina-stoyanova** [comment]
thanks, Jim! Merging & releasing now!
